common/src/test/java/tachyon/security/authentication/AuthenticationFactoryTest.java

```diff
@@ -135,8 +140,20 @@ public void simpleAuthenticationTest() throws Exception {
     client.open();
     Assert.assertTrue(client.isOpen());
 
+    // check user 'anyone'
+    verifyClientUser(client, "anyone");
+
+    // another connection by user 'anyone1'
+    TTransport client1 = createClient(mTachyonConf, "anyone1", "whatever");
```

Comment:
Not sure adding more coverage... Seems not.

Checking another username here was intent to ensure server could handle multiple client connection at the same time and maintain corresponding user info correctly.